packages/bpk-docs/src/pages/InfiniteScrollPage/InfiniteScrollPage.js

```diff
@@ -0,0 +1,169 @@
+/*
+ * Backpack - Skyscanner's Design Systrem
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import withInfiniteScroll, {
+  onItemsFetch,
+} from 'bpk-component-infinite-scroll';
+import BpkButton from 'bpk-component-button';
+import { BpkSpinner, SPINNER_TYPES } from 'bpk-component-spinner';
+
+import infiniteScrollReadme from '../../../../bpk-component-infinite-scroll/readme.md';
+
+import DocsPageBuilder from './../../components/DocsPageBuilder';
+import IntroBlurb from './../../components/neo/IntroBlurb';
+
+const elementsArray = [
+  'element 1',
+  'element 2',
+  'element 3',
+  'element 4',
+  'element 5',
+  'element 6',
+  'element 7',
+  'element 8',
+  'element 9',
+  'element 10',
+  'element 11',
+  'element 12',
+  'element 13',
+  'element 14',
+  'element 15',
+  'element 16',
+  'element 17',
+  'element 18',
+  'element 19',
+  'element 20',
+];
+
+const List = ({ elements }) => (
+  <div id="list">
+    {elements.map(element => (
+      <div key={element} style={{ height: '50px' }}>
+        {element}
+      </div>
+    ))}
+  </div>
+);
+
+List.propTypes = {
+  elements: PropTypes.arrayOf(PropTypes.any).isRequired,
+};
+
+const FixedHeighPanel = ({ children }) => (
+  <div style={{ height: '300px', overflow: 'scroll' }}>{children}</div>
+);
+
+FixedHeighPanel.propTypes = {
+  children: PropTypes.node.isRequired,
+};
+
+const InfiniteList = withInfiniteScroll(List);
+
+const components = [
+  {
+    id: 'infinite',
+    title: 'Infinite string list',
+    blurb: [],
```

Comment:
We should add a blurb in here such as 'The infinite scroll component allows long lists to be progressively loaded as items come into view.'

Reply:
Do we need one blurb for each example?